### ISSUE TITLE
Model training reproduction

### ISSUE THREAD

**Issue by Deurstann** [RainBowLuoCS/DiffusionTrack#15]
I am currently trying to reproduce the results of diffusiontrack on the test and ablation set of MOT17. But even though I followed all the steps described in the repository, I still obtain only 65.45 of MOTA on the test set of MOT17 and ~67.5 on the ablation set of MOT17. 
![Results by training a model for the test set](https://github.com/user-attachments/assets/3d64cec4-0b5b-4482-9271-bdd540d75c2d)
![Results by training a model for the ablation set](https://github.com/user-attachments/assets/032e99e8-bd13-4aaf-a264-a31b734c0cc6)
Are there other steps we have to do that are not described here and are the config files for the ablation and test set the same than what was done to produce the results we can see in the paper?

**Comment by RainBowLuoCS** [comment]
You need to load the official weights and carefully read the ablation experiment section in the paper to set the corresponding hyperparameters for best performance.

**Comment by Deurstann** [comment]
After reading again the paper and the code, I have found some inconsistencies:
- First in the paper it is said that the value used for nms3d threshold is 0.6 but in the code it is set by default to 0.7, what is the correct value that was used for the ablation result?
- Then in the paper it is said that the default parameters for the number of bounding box is 800 but in fact in the code it is 500 https://github.com/RainBowLuoCS/DiffusionTrack/blob/9714f01c21ff4524c4e78646c054bf8f8dbe8357/yolox/tracker/diffusion_tracker_kl.py#L182
- Again in the paper it is said that the default value for prior information usage is 25% but in the code the default value is 0. https://github.com/RainBowLuoCS/DiffusionTrack/blob/9714f01c21ff4524c4e78646c054bf8f8dbe8357/yolox/tracker/diffusion_tracker_kl.py#L178 If I understand correctly this parameter only control the number of copy of previous data so what does 25% means?

I am now closer to the results (72 of MOTA vs 73.6 in the paper for the ablation set), trying to find other parameters that could be different. If you have some information about that I would be very happy to know, thank you in advance for the help!

**Comment by RainBowLuoCS** [comment]
yes, you are right it may be a mistakes for me, all details can be found in the paper
